.github/workflows/build-electron.yml

```diff
@@ -40,6 +122,8 @@ jobs:
     
     - name: Upload installer artifacts
       uses: actions/upload-artifact@v2-preview
+      if: ${{ always() }}
```

Comment:
Curious why the conditional is needed here?

Reply:
@emlys , I hope this is not needed long-term, but there's an issue with a previous step where tests using the binaries fail even though those same binaries work well in the final package. So I wanted to upload the build artifacts no matter what. 

In the long-term we'll definitely want to get to the root of those test failures, but it's also quite possible some part of this build/test process will get refactored for other reasons, so I didn't care to get it perfect for this PR.